.circleci/config.yml

```diff
@@ -119,7 +120,7 @@ jobs:
       # Run EB deploy
       - run:
           name: Deploying
-          command: eb deploy --staged $EB_ENV_NAME
+          command: eb deploy --staged
```

Comment:
See below -- seems to me the ci/cd should handle deploying to the right place based on env (i.e. the values of `EB_ENV_NAME` etc.) and/or the branch name. Let me know if you want to discuss.